app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -282,6 +290,14 @@ class MainActivity : BaseActivity() {
         }
     }
 
+    private fun observeCanShowSyncDiscovery() {
+        syncDiscoveryManager.syncDiscoveryViewModel.canShowSyncDiscovery.observe(this) { canShowSyncDiscovery ->
```

Comment:
I find it odd to expose the manager's viewModel
Maybe we should wrap the value inside the manager ?

Reply:
Also, as the manager is a DefaultLifecycleObserver, we could directly put this observe inside it